apps/saleor-app-checkout/frontend/fetch/requests.ts

```diff
@@ -1,35 +1,42 @@
 import { PaymentProviderSettingsValues } from "@/saleor-app-checkout/types/api";
 import { FetchResponse } from "../hooks/useFetch";
-import { getAuthHeaders } from "../misc/auth";
 
 export interface PaymentProviderSettingsResult {
   data: PaymentProviderSettingsValues<"unencrypted">;
 }
 
 export const requestGetPaymentProviderSettings = ({
   saleorApiUrl,
+  token,
 }: {
   saleorApiUrl: string;
+  token: string;
 }): FetchResponse<PaymentProviderSettingsResult> =>
   fetch(
     `/api/payment-provider-settings/` + `?` + new URLSearchParams({ saleorApiUrl }).toString(),
     {
       method: "GET",
-      headers: getAuthHeaders(),
+      headers: {
+        Authorization: token ? `Bearer ${token}` : "",
+      },
```

Comment:
Why did we remove this function?

Reply:
Because it didn't really make any sense. It returned `{ Authorization: "Bearer ………" }` on the client and `{Authorization: ""}` on the server. It's confusing and against best practices.